packages/apps-config/src/links/subscan.ts

```diff
@@ -23,7 +23,8 @@ export default {
     Plasm: 'plasm',
     Polkadot: 'polkadot',
     Stafi: 'stafi',
-    Westend: 'westend'
+    Westend: 'westend',
+    Rococo: 'rococo'
```

Comment:
Thank you. Looks good, just needs to be moved alphabetical here, before Stafi (have not checked the CI failures, but I do believe that is why it is complaining)